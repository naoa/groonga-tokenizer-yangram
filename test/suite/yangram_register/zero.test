register tokenizers/yangram
yangram_register --ngram_unit 2 --split_alpha 0
tokenizer_list
tokenize TokenYaBigram "abc de" NormalizerAuto
