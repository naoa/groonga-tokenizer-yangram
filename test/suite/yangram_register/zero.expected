register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register --ngram_unit 2 --split_alpha 0
[[0,0.0,0.0],"TokenYaBigram"]
tokenizer_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "name": "TokenDelimit"
    },
    {
      "name": "TokenUnigram"
    },
    {
      "name": "TokenBigram"
    },
    {
      "name": "TokenTrigram"
    },
    {
      "name": "TokenBigramSplitSymbol"
    },
    {
      "name": "TokenBigramSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenBigramIgnoreBlank"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbol"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenDelimitNull"
    },
    {
      "name": "TokenYaBigram"
    }
  ]
]
tokenize TokenYaBigram "abc de" NormalizerAuto
[[0,0.0,0.0],[{"value":"abc","position":0},{"value":"de","position":1}]]
